src/autofill-utils.js

```diff
@@ -156,15 +157,19 @@ const safeExecute = (el, fn) => {
     const intObs = new IntersectionObserver((changes) => {
         for (const change of changes) {
             // Feature detection
+            // @ts-ignore todo: Why does TS not understand .isVisible here?
```

Comment:
It's a new version of the API only available in Chrome: https://web.dev/intersectionobserver-v2/.

Reply:
I'll update the comment

Reply:
Actually, we can add these 🥳